templates/components/account-menu.tpl

```diff
@@ -98,3 +98,9 @@
     </a>
   </div>
 {/block}
+
+{block name='account_link'}
+  <a class="btn btn-outline-primary hidden-on-desktop" href="{$urls.pages.my_account}">
```

Comment:
replace hidden-on-desktop by d-md-none
this .btn has to have an .mb-3 Its too close to the title

Reply:
@rodriciru We didn't do any styling at all, just the basic code and blocks now. :-)

.hidden-on-desktop is our own class, basically extending d-none and others. We can discuss if we want to use it or use the default boostrap ones.

Reply:
@NeOMakinG Maybe using reverting to `d-none d-md-block` /  `d-md-none` would bring more flexibility...

Reply:
i think that use classes wich do the same as bootstrap ones are redundant and unnecesary thats why i want to replace hidden-on-desktop class

Ok, i think .mb-3 class is layout as its write in .tpl classes

Reply:
Exactly! What's the point of all boostrap utility classes if you create your own for the same thing? That's why [I suggested](https://github.com/PrestaShop/theme-refacto/pull/121#issuecomment-975453760) refactoring theme structure. Almost whole theme can be made with just bootstrap. No need to make duplications when all it takes is using proper bootstrap class.

Reply:
Well, `hidden-on-desktop` and `hidden-on-mobile` extends Bootstrap utility classes, it's respecting CSS good practices as bootstrap is not an utility framework like Tailwind is, so we're trying to keep the paradigm between CSS and HTML. Using tailwind for example doesn't mean that you can't use custom class composed with utilities (https://tailwindcss.com/docs/extracting-components#extracting-component-classes-with-apply)

It's more readable, maintainable, avoid any responsive inconsistency because every breakpoints will be the same everywhere

@Hlavtox How would it bring more flexibility?

Reply:
> 

We're not creating classes to do the same thing, we're composing with bootstrap utilities in order to avoid redundancy, inconsistency...

Reply:
> 



> Am I really the only one that finds mixing BEM with kebab-case messed up? 😕

No you're not the only one, I think some components have been done before the BEM decisions, their refacto should be a WIP